data/com/mojang/realmsclient/util/RealmsUtil.mapping

```diff
@@ -0,0 +1,9 @@
+CLASS com/mojang/realmsclient/util/RealmsUtil
+	METHOD convertToAgePresentation (J)Ljava/lang/String;
+		ARG 0 time
+	METHOD convertToAgePresentationFromInstant (Ljava/util/Date;)Ljava/lang/String;
+		ARG 0 date
+	METHOD getTextures (Ljava/lang/String;)Ljava/util/Map;
+		ARG 0 uuid
+	METHOD uuidToName (Ljava/lang/String;)Ljava/lang/String;
+		ARG 0 uuid
```

Comment:
IMO it's a little unclear what UUID these methods take without looking at the source. I would suggest `profileUuid` or `playerUuid`.

Reply:
Good idea, I think `profileUuid` is better.